packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -557,8 +537,7 @@ class PaddleSliderValueIndicatorShape extends SliderComponentShape {
   static const double _twoSeventyDegrees = 3.0 * math.pi / 2.0;
   static const double _ninetyDegrees = math.pi / 2.0;
   static const double _thirtyDegrees = math.pi / 6.0;
-  static const Size preferredSize =
-      const Size.fromHeight(_distanceBetweenTopBottomCenters + _topLobeRadius + _bottomLobeRadius);
+  static const Size preferredSize = const Size.fromHeight(_distanceBetweenTopBottomCenters + _topLobeRadius + _bottomLobeRadius);
 
   static final Tween<double> _slideUpTween = new Tween<double>(begin: 0.0, end: 1.0);
```

Comment:
Is this tween really needed?   I think the value of `_slideUpTween.evaluate(activationAnimation)` is the same as `activationAnimation.value`

Reply:
Good point.  Removed.